packages/components/src/components/location-info/location-info.tsx

```diff
@@ -12,6 +12,11 @@ export class LocationInfo implements ComponentInterface {
      */
     @Prop() location;
 
+    /**
+     * @description Location's level property that can by translatable for specific languages. By default level value is 'Level'.
```

Comment:
Make the "would grandma understand this" test 😄 
The phrasing "Location's level property" is confusing.
How about

`The word used for "Level" when showing level info. Default is "Level".`